src/textual/app.py

```diff
@@ -822,27 +822,27 @@ def update_styles(self, node: DOMNode | None = None) -> None:
         self._require_stylesheet_update.add(self.screen if node is None else node)
         self.check_idle()
 
-    def mount(self, *anon_widgets: Widget, **widgets: Widget) -> AwaitMount:
-        """Mount widgets. Widgets specified as positional args, or keywords args. If supplied
-        as keyword args they will be assigned an id of the key.
+    def mount(self, *widgets: Widget) -> AwaitMount:
+        """Mount the given widgets.
+
+        Args:
+            *widgets (Widget): The widget(s) to mount.
 
         Returns:
             AwaitMount: An awaitable object that waits for widgets to be mounted.
-
         """
-        mounted_widgets = self._register(self.screen, *anon_widgets, **widgets)
-        return AwaitMount(mounted_widgets)
+        return AwaitMount(self._register(self.screen, *widgets))
```

Comment:
I'd opt to assign a variable here. It makes it easier to insert print statements and makes it self documenting. Not a blocker though!